### ISSUE TITLE
Prediction probability

### ISSUE THREAD

**Issue by Rafid013** [Xtra-Computing/thundersvm#132]
Hi,
For the same training parameters, there is a huge difference between the predictions of thundersvm (python) and scikit-learn. I have used standard scaler in both cases to scale the data. Accuracy was calculated using accuracy_score of sklearn. I also checked the probability values returned by both sklearn and thundersvm model which are different. The probability values are attached.

[thunder_proba.txt](https://github.com/Xtra-Computing/thundersvm/files/3007435/th_proba.txt)
[sklearn_proba.txt](https://github.com/Xtra-Computing/thundersvm/files/3007436/sk_proba.txt)

**Comment by zeyiwen** [comment]
Would you share your data set and python code with us? We need a minimum example to reproduce the problem and fix this issue.

**Comment by Rafid013** [comment]
No, it's okay. I made a very silly mistake. Nothing to do with thundersvm. Although I have another question, how does the predict_proba method return probabilities for binary classification? Is it exactly like sklearn (ordered classes, class 0 in first column, class 1 in second column)? Or does it depend on the order the classes appear in the training dataset?

**Comment by GODqinbin** [comment]
Hi, @Rafid013 It depends on the order the classes appear in the training dataset.

**Comment by Rafid013** [comment]
Thanks
